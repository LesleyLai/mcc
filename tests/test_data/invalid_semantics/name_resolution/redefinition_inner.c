int main(void)
{
  {
    int x = 10;
    int x = 42;
  }
}
