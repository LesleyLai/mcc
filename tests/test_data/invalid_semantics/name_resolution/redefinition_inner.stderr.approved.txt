{{filename}}:5:9: Error: redefinition of 'x'
5 |     int x = 42;
  |         ^

